cypress/integration/measurement.e2e.js

```diff
@@ -100,7 +100,7 @@ describe('Measurement Page Tests', () => {
         .should('have.attr', 'content', 'On August 7, 2020, Whatsapp was reachable in United States, explore more details and other measurements on OONI Explorer.')
     })
 
-    it.only('renders an unreachable og:description', () => {
```

Comment:
This should ideally be caught in CI by including `yarn lint` in the workflow. There are quite a few unaddressed lint errors (mostly missing prop-types) that would render it useless if enabled.